menuentryswapper/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -424,36 +317,6 @@ private void onMenuOpened(MenuOpened event)
 		{
 			String option = Text.removeTags(entry.getOption()).toLowerCase();
 
-			if (option.contains("trade with") && config.hideTradeWith())
-			{
-				continue;
-			}
-
-			if (option.contains("lookup") && config.hideLookup())
-			{
-				continue;
-			}
-
-			if (option.contains("report") && config.hideReport())
-			{
-				continue;
-			}
-
-			if (option.contains("examine") && config.hideExamine())
-			{
-				continue;
-			}
-
-			if (option.contains("net") && config.hideNet())
-			{
-				continue;
-			}
-
-			if (option.contains("bait") && config.hideBait())
-			{
-				continue;
-			}
```

Comment:
These are technically fine no?

Reply:
technically yes, removed in haste - ill add back